examples/example_apple_metal/main.mm

```diff
@@ -100,15 +110,11 @@ -(void)viewDidLoad
     self.mtkView.delegate = self;
 
 #if TARGET_OS_OSX
-    // Add a tracking area in order to receive mouse events whenever the mouse is within the bounds of our view
-    NSTrackingArea *trackingArea = [[NSTrackingArea alloc] initWithRect:NSZeroRect
-                                                                options:NSTrackingMouseMoved | NSTrackingInVisibleRect | NSTrackingActiveAlways
-                                                                  owner:self
-                                                               userInfo:nil];
-    [self.view addTrackingArea:trackingArea];
-
     ImGui_ImplOSX_Init(self.view);
 
+    [NSApp activateIgnoringOtherApps:YES];
```

Comment:
Could you describe purpose of this and other changes in this file? Few changes are needed for viewports support, what about everything else?

Reply:
We shouldn't need that – it can be removed.